optimum/habana/accelerate/accelerator.py

```diff
@@ -75,6 +75,10 @@
     GaudiDynamoBackend,
     GaudiFullyShardedDataParallelPlugin,
     GaudiTorchDynamoPlugin,
+    te_forward_convert,
```

Comment:
Function names are a little confusing

Reply:
I agree, but on the other hand I feel like this has more to do with the Transformer Engine library which pushed the use of `te`. So I think it's okay since it will always be used in the context of fp8 mixed precision.